Dockerfile.compute-node

```diff
@@ -50,15 +50,14 @@ RUN apt update && \
     libcgal-dev libgdal-dev libgmp-dev libmpfr-dev libopenscenegraph-dev libprotobuf-c-dev \
     protobuf-c-compiler xsltproc
 
-RUN wget https://gitlab.com/Oslandia/SFCGAL/-/archive/v1.3.10/SFCGAL-v1.3.10.tar.gz && \
```

Comment:
Look at the last step. Finally, we're installing this lib via `apt-get` for the final image. We must be compatible. Have you tested it?

Reply:
Added a comment about it